docs/orchestration/tutorial/first.md

```diff
@@ -37,6 +37,23 @@ flow.register(project_name="Hello, World!")
 Registration only sends data about the existence and format of your flow; **no actual code from the flow is sent to the Prefect API**. Your code remains safe, secure, and private in your own infrastructure!
 :::
 
+:::tip Deduplicating registration calls
```

Comment:
I do not love this as a location but there are links saying "learn more about flow registration" that point here. Having a discussion of flow registration and best practices would make sense outside of this tutorial (probably just in concepts) but is out of the scope of this PR.